addons/web/static/tests/core/utils/autoresize.test.js

```diff
@@ -72,8 +73,31 @@ test(`call onResize callback`, async () => {
         }
     }
     await mountWithCleanup(ResizableInput);
-    expect.verifySteps(["onResize"]);
+    expect.verifySteps(["onResize", "onResize"]);
 
     await contains(`.resizable-input`).edit("new value", { instantly: true });
-    expect.verifySteps(["onResize"]);
+    expect.verifySteps(["onResize", "onResize"]);
```

Comment:
I'm not in love with this. The resizeObserver will call resize a second time when the input changes dimensions due to the user typing things. This is no big matter and I haven't found a fix for it so we can merge as is unless you find a quick working solution (note that passing the test without this diff is no sufficient proof that it works in practice).